src/main/java/com/hcl/appscan/cli/handlers/InvokeDynamicScan.java

```diff
@@ -342,7 +353,10 @@ private  Optional<ScanResults> runScanAndGetResults() throws Exception {
 
             }
 
-        }catch (Exception e) {
+        }catch (ParameterException pe){
```

Comment:
@mattmurp - Agreed. For ParameterException  , the idea was to let it propagate further to the calling methods till the main method so that Picoli can handle it in a specific manner. For other types of exceptions (caught by Exception class) , I am using a generic logger to log it (which attaches a timestamp to the log). For Parameter Related exceptions , Parimal had raised a few bugs where the ask was to remove the timestamp in the error message and it should be uniform with other ParameterExceptions which were handled by Picoli., Hence I was catching the  ParameterException  and throwing it , so that Exception catch block will not catch and log it.
Sure , There is a better way to handle it but is it fine If I refactor this later so that we can speedify the release..